src/app/forms/templates/general/hgv-trl-brakes.template.ts

```diff
@@ -1,8 +1,8 @@
 import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
 
-export const TrlBrakes: FormNode = {
-  name: 'trlBrakesSection',
+export const HgvAndTrlBrakesTemplate: FormNode = {
+  name: 'hgvAndTrlBrakesSection',
   label: 'Brakes',
```

Comment:
brakeActuator and leverLength need NUMBER edit type adding to the form group.

Reply:
antiLockBrakingSystem also needs to be required

Reply:
It seems that brakeActuator and leverLength don't have validation on them though even on develop which they should 👀

Reply:
Are they making sure that it sends the numbers as a number type and not a string?

Reply:
Yep